pkg/up/up.go

```diff
@@ -680,7 +727,7 @@ func (u *upRunner) run() error {
 			break
 		}
 	}
-	fmt.Printf("pods ready (%d/%d)\n", len(u.apps), len(u.apps))
+	fmt.Printf("pods ready (%d/%d)\n", len(u.appsThatNeedToBeReady), len(u.appsThatNeedToBeReady))
```

Comment:
I believe there is another log statement which prints a similar line, but that one has not been updated.

Reply:
IMO the similar check of Services cannot be applied here. As we are trying to create a pod and pod returns error message immediately. Further down in the code, we try to receive the message from the pod status channel and channel never sends any message.